core/src/main/java/org/jboss/pnc/rex/rest/TaskEndpointImpl.java

```diff
@@ -76,4 +76,13 @@ public Response cancel(String taskID) {
 
         return Response.accepted().build();
     }
+
+    @Override
+    @Retry
```

Comment:
You have to wire the @Retry in the application.yaml (look at the bottom) so that the internal retries are configured. 
**BUT** locally, I've reworked FaultTolerance for internal Transactions so this is not needed anymore. If you're willing to wait, it should definitely be out this week together with fix for Alignment in Orch.

Reply:
Sure can wait.

Reply:
So, in those places where you want `@Retry`, just add `@ApplyFaultTolerance("internal-retry")` viz https://github.com/project-ncl/rex/blob/ed5bb53eb3d309d8cdf72628e2e9e2474c8b3bf4/core/src/main/java/org/jboss/pnc/rex/core/delegates/TolerantQueueManager.java#L44